projects/aem-angular-core-wcm-components/core/src/emptyfunctions/language-navigation.v1.isempty.ts

```diff
@@ -0,0 +1,12 @@
+import {NavigationV1Item, NavigationV1Model} from "./navigation.v1.isempty";
+
+export interface LanguageNavigationV1Item extends NavigationV1Item{
```

Comment:
This one doesn't have IsEmptyFn ?

Reply:
This one just reuses the navigation one, as it's identical